contrib/zmq/zmq_sub.py

```diff
@@ -1,43 +1,70 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 # Copyright (c) 2014-2016 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
```

Comment:
nit: I'd like to see a module-level docstring describing what this does (which can include the comment block below about the pre-2.7 version)